content/en/docs/ibm/using-icr.md

```diff
@@ -0,0 +1,80 @@
++++
+title = "Using IBM Cloud Container Registry (ICR)"
+description = "Instructions for using private images from ICR"
+weight = 15
++++
+
+This guide describe steps for use cases involving private container images by using IBM Cloud Container Registry (ICR).
+
+## Prerequisites
+
+* Install and configure the [IBM Cloud CLI](https://cloud.ibm.com/docs/cli?topic=cli-getting-started).
+* Install the CLI plug-in for the IBM Cloud Container Registry by running the command `ibmcloud plugin install container-registry`.
+* Create a namespace in ICR with command `ibmcloud cr namespace-add <my_namespace>`, replace `<my_namespace>` with your preferred name.
+
+**Note**: the [ICR namespace](https://cloud.ibm.com/docs/Registry?topic=Registry-getting-started#gs_registry_namespace_add) is different from the Kubeflow Profile namespace. The ICR namespace is used to group container images stored in ICR while a [Kubeflow Profile](/docs/components/multi-tenancy/overview/) namespace is a grouping of all Kubernetes clusters onwed by a user.
+
+## Image pull secret
+
+An image pull secret is required to to pull container images from IBM Cloud Container Registry. You can use the default image pull secret set up by the cluster or use your account's IAM API key.
+
+### Use default image pull secret
+
+By default, the IBM Cloud Kubernetes cluster is set up to pull images from only your account's namespace in IBM Cloud Container Registry by using the secret `all-icr-io` in the `default` namespace. A cluster admin can copy this secret to any Kubernetes namespace used as Kubeflow profile. For example, run below command to copy the secret `all-icr-io` to the `anonymous` namespace:
+
+```
+kubectl get secret all-icr-io -n default -o yaml \
+| sed 's/namespace: default/namespace: anonymous/g' \
+| kubectl -n anonymous create -f -
+```
+
+Once this secret is ready in your Kubeflow profile, A data scientist can use this secret to pull container images from ICR.
+
+See details and FAQs from the official guide [Setting up an image registry](https://cloud.ibm.com/docs/containers?topic=containers-registry).
+
+### Use IAM API Key
+
+You will need to use an IBM Cloud IAM API Key to work with ICR in below cases:
+1. has no access to the default image pull secret `all-icr-io` from the `default` namespace.
```

Comment:
Thanks! I use `1,1,1` since the number doesn't have to be in number order. see https://www.markdownguide.org/basic-syntax#ordered-lists

> To create an ordered list, add line items with numbers followed by periods. The numbers don’t have to be in numerical order, but the list should start with the number one.